### PR TITLE
Opcode Updates

### PR DESCRIPTION
**Pull request by dio85**
Fixed Duplicated

### CODE REVIEW

**Comment by DrEhsan** [comment]
fix the tabs
also you have enabled some opcodes , what about their structures? have you checked them?
and what about their responses? enabling an opcode without response is useless.

**Comment by SkyFire** [comment]
This will need more review. BTW What core are you copy/pasting this from.

File: src/server/game/Server/Protocol/Opcodes.cpp [hunk 1]
```diff
@@ -751,6 +750,7 @@ void OpcodeTable::InitializeServerTable()
     DEFINE_OPCODE_HANDLER(SMSG_AREA_TRIGGER_MESSAGE,               0x0000, STATUS_UNHANDLED);
     DEFINE_OPCODE_HANDLER(SMSG_AREA_TRIGGER_MOVEMENT_UPDATE,       0x0000, STATUS_UNHANDLED);
     DEFINE_OPCODE_HANDLER(SMSG_ARENA_ERROR,                        0x0000, STATUS_UNHANDLED);
+	DEFINE_OPCODE_HANDLER(SMSG_ARENA_PREP_OPPONENT_SPECIALIZATIONS, 0xBE, STATUS_NEVER     ); // ADDED
```
Comment (SkyFire) [inline src/server/game/Server/Protocol/Opcodes.cpp]: // 5.4.8 18414
There all over the place. It just looks sloppy and untested.

**Comment by dio85** [comment]
A working core....

**Comment by SkyFire** [comment]
A working core? Without structures?

**Comment by dio85** [comment]
no, but if i write all, nobody have work. this just little help but if not need i can delete.

**Comment by RAlexis** [comment]
Mate, post the structures alongside the opcodes if you really help them. It helps us all.

File: src/server/game/Server/Protocol/Opcodes.cpp (hunk 1 above)
Comment (AriDEV) [inline src/server/game/Server/Protocol/Opcodes.cpp]: Tabs Here Tabs There ... 0xBE ...

**Pull request closed without merging**